Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.10-alpine as builder
+
+RUN apk --update upgrade \
+&& apk --no-cache --no-progress add git mercurial bash gcc musl-dev curl tar \
+&& rm -rf /var/cache/apk/*
+
+WORKDIR /go/src/github.com/letsencrypt/pebble
+COPY . .
+
+RUN go get -u github.com/jmhodges/clock \
+&& go get -u gopkg.in/square/go-jose.v2
+
+RUN go install ./...
+
+## main
+FROM alpine:3.7
+
+RUN apk update && apk add --no-cache --virtual ca-certificates
+
+COPY --from=builder /go/bin/pebble /usr/bin/pebble
+COPY --from=builder /go/src/github.com/letsencrypt/pebble/test/ /test/
+
+ENTRYPOINT [ "/usr/bin/pebble" ]
```

Comment:
I think this needs a `--config /test/config.json` no? Otherwise I think `pebble` will default to trying to read from `test/config/pebble-config.json` in the pwd.

Reply:
Ahh, and PWD == `/` in this case so everything works :+1: 

Thanks for clarifying.

Reply:
This should probably be `CMD` rather than `ENTRYPOINT`. The difference: If you pass an alternate command to `docker run`, it gets passed as an argument to `ENTRYPOINT`, but it overrides `CMD`. So for instance if you wanted to run `bash` in the context of this container it would not work right due to the `ENTRYPOINT`.